lib/dish.rb

```diff
@@ -0,0 +1,7 @@
+class Dish
+  attr_reader :name, :category
+  def initialize(name, category)
+    @name = name
+    @category = :appetizer || :entree || :dessert
```

Comment:
Try not to get hung up on the datatype that is getting passed into a class - symbols can be used just like strings or integers: `@category = category`